mmrazor/models/distillers/configurable_distiller.py

```diff
@@ -38,6 +38,14 @@ class ConfigurableDistiller(BaseDistiller):
         distill_deliveries (dict, optional): Config for multiple deliveries. A
             distill algorithm may have more than one delivery. Defaults to
             None.
+        student_connectors (dict, optional): Config for multiple connectors. A
```

Comment:
Explain the key mapping relations between connectors, distill_losses, and loss_forward_mappings in ``Note`` below.

Reply:
The key mapping relations should be checked. Refers to `_check_loss_forward_mappings`